parsec/utils/mca_param_cmd_line.c

```diff
@@ -41,30 +41,23 @@ static void add_to_env(char **params, char **values, char ***env);
 
 
 /*
- * Add -mca to the possible command line options list
+ * Add -parsec to the possible command line options list
  */
 int parsec_mca_cmd_line_setup(parsec_cmd_line_t *cmd)
 {
     int ret = PARSEC_SUCCESS;
 
-    ret = parsec_cmd_line_make_opt3(cmd, '\0', "mca", "mca", 2,
-                                  "Pass context-specific MCA parameters; they are considered global if --gmca is not used and only one context is specified (arg0 is the parameter name; arg1 is the parameter value)");
-    if (PARSEC_SUCCESS != ret) {
-        return ret;
-    }
-
-    ret = parsec_cmd_line_make_opt3(cmd, '\0', "gmca", "gmca", 2,
```

Comment:
Don't see why we need gmca since we are not mpiexec. Let me know if there is a point.

Reply:
I restored the gmca functionality, it may be useful for the info params, so I'll remove it if it turns out useless then.